lib/github/cache.nit

```diff
@@ -58,7 +58,7 @@ redef class GithubAPI
 	redef fun get(key, headers, data) do
 		if not enable_cache then return super
 		if store.has_key(key) then
-			message(1, "Get {key} (cache)")
+			# print "Get {key} (cache)" # debug
```

Comment:
Seriously ? Replacing old logging infrastructure with greasy commented hard coded prints. I like that!

Reply:
Next step is to kill `json_store`.

If I hit it hard enough, it should take `github_cache` down with it.